tests/test_api_auth.py

```diff
@@ -238,10 +229,6 @@ def test_02_auth_without_split(self):
                 self.assertEqual(4031, result['error']['code'], result)
                 self.assertEqual('Authentication failure. Wrong credentials',
                                  result['error']['message'], result)
-            if six.PY2:
-                expected = "The user User(login=u'cornelius@realm1', " \
-                           "realm=u'realm1', resolver='') exists in NO resolver."
-            else:
                 expected = "The user User(login='cornelius@realm1', " \
```

Comment:
here also

Reply:
fixed in a1e752d6c5abb37fab930e65b3c64b3680e3f3f1.